Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonToastNotificationManager.cs

```diff
@@ -0,0 +1,17 @@
+﻿#region BSD License
+/*
+ *
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
+ *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2023 - 2023. All rights reserved.
+ *
+ */
+#endregion
+
+namespace Krypton.Toolkit
+{
+    [ToolboxItem(true)]
+    [ToolboxBitmap(typeof(KryptonToastNotificationManager), @"ToolboxBitmaps.KryptonInputBox.bmp")]
+    public sealed class KryptonToastNotificationManager : Component
```

Comment:
Do not make them `sealed`, as this prevents others from deriving and adding extra functionality (i.e. Ext code base)

Reply:
Done